api/libproject/devstart.rb

```diff
@@ -814,24 +824,25 @@ def deploy(cmd_name, args)
   common.run_inline %W{
     gcloud app deploy
       build/staged-app/app.yaml
-      build/staged-app/WEB-INF/appengine-generated/cron.yaml
-      --project #{gcc.project} #{promote}
-  } + (op.opts.quiet ? %W{--quiet} : []) + (op.opts.version ? %W{--version #{op.opts.version}} : [])
+  } + (op.opts.deploy_cron ? %W{build/staged-app/WEB-INF/appengine-generated/cron.yaml} : []) +
+    %W{--project #{gcc.project} #{promote}} +
+    (op.opts.quiet ? %W{--quiet} : []) +
```

Comment:
I will accept the risk of splitting this into two lines... Seemed to work fine locally (+ on the previous line).